src/counts.jl

```diff
@@ -61,6 +81,18 @@ proportions(x::IntegerArray, k::Integer, wv::WeightVec) = proportions(x, 1:k, wv
 proportions(x::IntegerArray) = proportions(x, span(x))
 proportions(x::IntegerArray, wv::WeightVec) = proportions(x, span(x), wv)
 
+"""
+    proportions(x::IntegerArray, levels::IntUnitRange, wv::WeightVec)
+
+### Args:
+* `x`: An `IntegerArray`.
+* `levels`: An integer unti range.
+* `wv`: An optional weight vector.
+
+Compute the proportions of values in a:b with respect to x.
```

Comment:
Severals problems here:
- what is `a:b`?
- "with respect to x" isn't obvious
- you should put optional argument between `[ ]` (BTW, `levels` is optional too)
- typo in "unti"
- see also the points raised above